src/modules/account/components/MultisigComponent.vue

```diff
@@ -100,20 +117,38 @@ export default {
       return cosignerAccountsList
     }, {deep: true} )
 
+    const strPublicKey = computed(()=>{
+      let strPublicKey = props.PublicKey;
+      return  strPublicKey
+    })
+
```

Comment:
Props are immutable, this function is unnecessary, can just use props.publicKey

Reply:
Noted

Reply:
i mean you dont need this function at all, you can use strPublicKey on html and props.strPublicKey in script